### ISSUE TITLE
formatting issues

### ISSUE THREAD

**Issue by skolenik** [fcps/research-request#1]
1. Please provide the README.md file.
2. In that README.md, you can provide a reference to the reader for the HTML files -- https://htmlpreview.github.io/ is a popular option.

Thanks.

**Comment by wbuchanan** [comment]
@skolenik 
I'm not sure how to provide the reference for the reader, but we used [StackEdit](https://github.com/benweet/stackedit) as we were putting things together and modified some of the CSS styling manually to make the output HTML work well with out website CMS.

**Comment by wbuchanan** [comment]
@skolenik 
Well, all of the markdown is there now.  If you copy the raw text over into [StackEdit](https://stackedit.io/) it should render relatively similar to what we have published.  The differences are that I used `flex-box` CSS elements to contain the workflow diagram and corresponding text related to the diagram so it would display side-by-side on larger screens and vertically on smaller screens.  I also added some overrides to some of the CSS styling used by [StackEdit](https://stackedit.io/) to make sure the text and all that satisfied the § 508/WCAG requirements for accessibility purposes.
